tools/download_ipk.sh

```diff
@@ -22,12 +23,12 @@ download() {
     IPK_URL="https://gitlab.com/prpl-foundation/prplmesh/-/jobs/artifacts/$BRANCH/raw/build/$DEVICE/prplmesh.ipk?job=build-for-$DEVICE"
     BUILDINFO_URL="https://gitlab.com/prpl-foundation/prplmesh/-/jobs/artifacts/$BRANCH/raw/build/$DEVICE/prplmesh.buildinfo?job=build-for-$DEVICE"
     # curl options - fail on error (-f), follow redirect (-L)
-    curl -f -L "$IPK_URL" --output "$IPK_PATH"/prplmesh.ipk
-    curl -f -L "$BUILDINFO_URL" --output "$IPK_PATH"/prplmesh_buildinfo.txt
+    curl -f -L "$IPK_URL" --output "$IPK_PATH"/prplmesh.ipk "$REDIRECT"
```

Comment:
As mentioned elsewhere, the non-verbose redirect will not work due to the quotes.

However, specfically for curl, you should use `-s` instead of redirecting output. Also, you probably *do* want to keep errors, so it's better to use `-s -S`.
```suggestion
    curl ${VERBOSE:+ -s -S} -f -L "$IPK_URL" --output "$IPK_PATH"/prplmesh.ipk
```

Note the optional argument handling suggestion from [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086). Note that this means VERBOSE needs to be defined as empty instead of false by default, so if you want to apply this trick in any script that uses functions.sh, it's a bigger change.

Reply:
Unfortunately, we need the opposite - if `VERBOSE` **is** defined not to add the `-s -S` options. So it has to be `--quite` instead, no?